vignettes/FDZ_Data_Cleaning.Rmd

```diff
@@ -166,8 +166,8 @@ extractMeta(dat7, vars = c("books"))
 
 
 ## Anonymization: Form grouping variable
-For anonymization reasons, we partially form grouping variables for items with a case number <6. To do this, we create a new _FDZ variable by "cloning" the variable we want to group with `cloneVariable()`. For `varName` we enter the original variable name, for `new_varName` we enter the variable name including the appendix "_FDZ". The variable label is automatically adjusted with the option `label_suffix`.
-With `recodeGADS()` the old values (`oldValues`) are replaced by new ones (`newValues`). With the help of `changeValLabels()` the value labels of the new groupings are changed.
+Small case numbers pose a risk of anonymity. For variables with a number of cases <6, which are not ratings, feelings or opinions, it is advisable to combine small numbers of cases into meaningful groups. To do so, `cloneVariable()` creates a new _FDZ variable by cloning the variable to be grouped. For `varName` the original variable name is entered, for `new_varName` the variable name including the appendix "_FDZ". The variable label is automatically adjusted with the option `label_suffix`.
```

Comment:
I like the wording a lot! Could it emphasized, however, that such pseudonymization is only applied to variables that contain sensitive information about individuals? (German term: Personenbezug)

Reply:
thanks! will do.